src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -181,13 +184,15 @@ function MdAutocomplete () {
               ng-if="$mdAutocompleteCtrl.loadingIsVisible()"\
               md-mode="indeterminate"></md-progress-linear>\
           <md-virtual-repeat-container\
+              id="{{menuContainerId}}"\
```

Comment:
This looks to be left over from the previous implementation and should be removed.